openbas-api/src/main/java/io/openbas/service/MapperService.java

```diff
@@ -176,4 +177,34 @@ private void updateInjectImporter(List<InjectImporterUpdateInput> injectImporter
     });
   }
 
+  public List<ImportMapperAddInput> exportMappers(List<String> idsToExport) {
```

Comment:
I don't understand why we need to recreate an object here

Reply:
It's kind of a shortcut : instead of exporting objects that match what is in the database, we export the object that would allow us to recreate the same mapper (which is why the import code is so small). This is why we're converting it into ImportMapperAddInput.
It result in a smaller file but to be honest, I'm kind of unsure if that's the best way to do it as it assumes that the export is made so that it can be imported. What do you think of it ?

Reply:
We need to talk about it.
We have an existing mechanism to export based on Json annotation. We can challenge this too to check if there are a modern way to do it.